server/src/entity/arrow.rs

```diff
@@ -0,0 +1,65 @@
+use shrev::EventChannel;
+use specs::{
+    Component, Entity, Read, ReadStorage, ReaderId, System, SystemData, VecStorage, World,
+};
+
+use feather_core::{Item, Position};
+
+use crate::entity::NamedComponent;
+use crate::player::PLAYER_EYE_HEIGHT;
+use crate::util::Util;
+
+/// Component for arrow entities.
+#[derive(Default)]
+pub struct ArrowComponent;
+
+impl Component for ArrowComponent {
+    type Storage = VecStorage<Self>;
```

Comment:
For marker components (those with zero size), `NullStorage` should be used.

https://slide-rs.github.io/specs/05_storages.html